platform/docs/docs/platform/services/data/StateSyncService.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 4
+sidebar_label: State Sync Service
+---
+
+# State Sync Service
+
+## Overview
+The state sync service is designed to allow short and long term memory of things such as
+annotations applied, last annotation state, hanging protocol viewport state,
+window level etc.  This allows for better interaction with things like navigation
+between hanging protocols, ensuring that the previously displayed layouts
+can be redisplayed after returning to a given hanging protocol.
+
+Currently there are only two configured states, the mode based ones and the
+application lifetime states.  The intent is to allow other lifetime basis
+to be configured, for example, browser local store lifetime.
+
+## Events
+
+Currently the service does not fire events.
```

Comment:
In the code there is a MODIFIED event. If it is not actually used, please remove it.